doc/src/manual/workflow-tips.md

```diff
@@ -44,8 +43,14 @@ line. A common pattern includes the following elements:
     The advantage is that you can now do `using Tmp` in your test code and can therefore avoid prepending
     `Tmp.` everywhere. The disadvantage is that code can no longer be selectively copied to the REPL
     without some tweaking.
-  * **Lather. Rinse. Repeat.** Explore ideas at the `julia` command prompt. Save good ideas in `tst.jl`.
-
+    
+  * **Lather. Rinse. Repeat.** Explore ideas at the `julia` command prompt. Save good ideas in `tst.jl`. Occasionally restart the REPL, issuing
+  
+    ```
+    reload("Tmp")
```

Comment:
reload is deprecated so should likely not be recommended.

Reply:
I did not know that! I was using it all the time.

Reply:
Where is the PR/issue where `reload` was deprecated? I want to read it.

Reply:
Only temporarily removed, it should be reinstated (#25720). Although, in most cases, a Revise-based workflow is much faster and more convenient now.